packages/fxa-auth-db-mysql/lib/db/schema/patch-114-115.sql

```diff
@@ -0,0 +1,54 @@
+--
+-- This migration introduces a table to track sent emails related to
+-- subscriptions.
+--
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('114');
+
+CREATE TABLE IF NOT EXISTS sentSubscriptionEmails (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  uid BINARY(16) NOT NULL,
+  emailType VARCHAR(60) NOT NULL,
+  subscriptionId VARCHAR(255) NULL,
+  planId VARCHAR(255) NULL,
```

Comment:
We might want to send emails not about a particular subscription. Also, planId was replaced by priceId. Not sure we want to hard-code these into table columns as Stripe changes them.

Reply:
Maybe we should use a JSON column type to add arbitrary email argument values of distinction for more flexibility.

Reply:
> 
> 
> We might want to send emails not about a particular subscription. 

Yes, which is why both of those columns are nullable.

> Also, planId was replaced by priceId. Not sure we want to hard-code these into table columns as Stripe changes them.

That was a decision to keep things consistent as we still use "plan" in most places.